user-service/src/main/java/ru/vsu/csd/userservice/UserServiceApplication.java

```diff
@@ -6,8 +6,8 @@
 @SpringBootApplication
```

Comment:
Seems like u opened this in another text editor and it edited everything with 1 tab == 8 spaces. Make 1 tab = 4 spaces pls in all files